packages/fill-scenarios/tslint.json

```diff
@@ -1,3 +1,6 @@
 {
-    "extends": ["@0xproject/tslint-config"]
+    "extends": ["@0xproject/tslint-config"],
+    "rules": {
+        "no-unused-variable": false
```

Comment:
Why not add it as a base rule?

Reply:
It is a base rule but I disabled it for this package because I noticed there were generated files. Maybe I should just ignore that directory?

For example `dummy_token.ts` 
/**
 * This file is auto-generated using abi-gen. Don't edit directly.
 * Templates can be found at https://github.com/0xProject/0x-monorepo/tree/development/packages/contract_templates.
 */

Reply:
Yeah, we should ignore generated files. I thought I saw @LogvinovLeon do this in a recent PR.

Reply:
I've gone ahead and excluded generated contract wrapper files from the linter.